src/matrixrtc/CallMembership.ts

```diff
@@ -184,10 +196,24 @@ export class CallMembership {
         }
     }
 
+    /**
+     * @returns The number of milliseconds until the membership expires or undefined if applicable
+     */
     public getMsUntilExpiry(): number | undefined {
-        if (isLegacyCallMembershipData(this.membershipData)) return this.getLocalExpiry()! - Date.now();
+        if (isLegacyCallMembershipData(this.membershipData)) {
+            // Assume that local clock is sufficiently in sync with other clocks in the distributed system.
+            // We used to try and adjust for the local clock being skewed, but there are cases where this is not accurate.
+            // The current implementation allows for the local clock to be -infinity to +MatrixRTC.MEMBERSHIP_EXPIRY_TIME/2 (i.e. +30mins).
```

Comment:
```suggestion
            // The current implementation allows for the local clock to be -infinity to +MatrixRTCSession.MEMBERSHIP_EXPIRY_TIME/2 (i.e. +30mins).
```
Plus, maybe we should add a comment in `MatrixRTCSession.membershipEventNeedsUpdate` with a reminder to update this comment if the expiry calculation were to change & thus make this comment stale (like if `MEMBERSHIP_EXPIRY_TIME` were to change from 30 mins to some other value).

Reply:
I've stripped the `i.e.` instead.